x-pack/plugin/ml/build.gradle

```diff
@@ -100,6 +101,7 @@ task internalClusterTest(type: RandomizedTestingTask,
                          dependsOn: unitTest.dependsOn) {
   include '**/*IT.class'
   systemProperty 'es.set.netty.runtime.available.processors', 'false'
+  systemProperties 'java.security.policy': file("src/main/plugin-metadata/plugin-security-test.policy").absolutePath
```

Comment:
I needed to add these two lines for the integration tests to run. Apparently, with how the internal cluster is built, the painless plugin is not included, so I included it, and add the requisite java security policy so that it can run. 

If there is a better way (or if the tests are deemed unnecessary), then I will happily change this.